### PR TITLE
Expose highlighAtomColors, highlighBondColors and highlightAtomRadii to CFFI and JS

### PR DESCRIPTION
**Pull request by ptosco**
This PR addresses https://github.com/rdkit/rdkit-js/issues/212.

### CODE REVIEW

File: Code/MinimalLib/tests/tests.js [hunk 1]
```diff
@@ -992,6 +992,40 @@ M  END
...
+    }));
+    console.log(`***SVG***\n${svg}`);
+    /*
+    assert(svg.includes('fill:#FF7F7F'));
+    assert(svg.includes('fill:#FF0000'));
+    assert(svg.includes('fill:#00FF00'));
+    assert(svg.includes('fill:#00B2E5'));
+    assert(svg.includes("width='127px'"));
+    assert(svg.includes('</svg>'));
+    */
```
Comment (greglandrum) [inline Code/MinimalLib/tests/tests.js:1026]: You aren't actually testing anything here. Don't you want to un-comment these `assert`s?

**Review by greglandrum — CHANGES_REQUESTED** [review]
Some tests are commented out, but otherwise this looks fine.

File: Code/MinimalLib/tests/tests.js (hunk 1 above)
Comment (ptosco) [reply Code/MinimalLib/tests/tests.js]: Of course :/ Sorry, I forgot to uncomment them before committing.

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**